src/SpeedInput/SpeedInput.tsx

```diff
@@ -0,0 +1,110 @@
+import { FC, useState, ChangeEvent } from 'react';
+import RadioGroup from 'src/RadioGroup/RadioGroup';
+import MinMaxInput from 'src/Filter/modules/DurationFilter/MinMaxInput';
+import { RadioGroupWrapper } from 'src/SpeedInput/speedInputStyles';
+
```

Comment:
Import theme and theme provider as we used in other components.
import { ThemeProvider } from '@emotion/react';
import theme from 'src/styles/theme';